src/commcare_cloud/ansible/roles/elasticsearch/tasks/main.yml

```diff
@@ -126,6 +132,16 @@
     - cron
     - backups
 
+- name: Create initial snapshot
```

Comment:
I'm not sure this should be part of this role as it would get executed every time you run the role

Reply:
Running it again does not impact the snapshot. This is the default behavior we are using in S3_backups also.